paperqa/docs.py

```diff
@@ -766,19 +776,22 @@ async def aquery(  # noqa: PLR0912
             )
         else:
             with set_llm_session_ids(session.id):
-                answer_result = await llm_model.run_prompt(
-                    prompt=prompt_config.qa,
-                    data={
-                        "context": context_str,
-                        "answer_length": answer_config.answer_length,
-                        "question": session.question,
-                        "example_citation": prompt_config.EXAMPLE_CITATION,
-                    },
+                data = {
+                    "context": context_str,
+                    "answer_length": answer_config.answer_length,
+                    "question": session.question,
+                    "example_citation": prompt_config.EXAMPLE_CITATION,
+                }
+                messages = [
+                    Message(role="system", content=prompt_config.system),
+                    Message(role="user", content=prompt_config.qa.format(**data)),
```

Comment:
If we only use `data` once, it's extra indirection to store a `dict` then use `**` expansion in the next line. Can we use normal kwargs

```python
content=prompt_config.qa.format(
    context=context_str,
    answer_length=answer_config.answer_length,
))
```

Reply:
Can you still do this one? I don't think we need the `data` local